### DISCUSSION TITLE
Card: Maastrich Treaty

### DISCUSSION THREAD

**Discussion by RebelusQuo** [RebelusQuo/the-wall-is-down#4] [Ideas]
- Clarification: Is it "up to" in the case that US performs the Event?
- No, it should be 1 influence always, even if played by US. EU has a very hard play with a very long border and several countries, it needs these support cards.
- The phrase in the card text is ".. in up to two countries among .." so the "up to" phrase relate to the selection of countries, not the amount and kind of Influence. Given your response I take it US should have to add Influence to as many countries as possible so another reason to remove the "up to" phrase?

**Comment by rodrigoSantamaria** [comment]
Yes, definitely. 1 influence in 2 countries, always. The only (additional) thing that should be clear is that you cannot use it to add 2 influence to a single country.

**Comment by RebelusQuo** [comment]
Ok, with that we have the following suggestion:
> EU gains 1 VP.
> Add 1 EU Influence in each of two countries among Germany, France, Italy, Spain/Portugal, Benelux, Denmark, and Greece.

**Reply by rodrigoSantamaria** [reply]
Perfect :)
